content/chat/connect.textile

```diff
@@ -24,6 +25,9 @@ A connection can have any of the following statuses:
 blang[javascript].
   Use the "@current@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.ConnectionStatus.html#current property to check which status a connection is currently in:
 
+blang[swift].
+  Use the "@status@":https://sdk.ably.com/builds/ably/#status property to check which status a connection is currently in:
```

Comment:
This appears to be a dead link right now?

Reply:
I'm going to insert links that will look like:
```
"@status()@":https://sdk.ably.com/builds/ably/ably-chat-swift/main/typedoc/interfaces/chat.Rooms.html#status
```
Which are also yet dead, but with a potential to go live as soon as swift docs generation PR merged (https://github.com/ably/ably-chat-swift/pull/165)

Reply:
updated